2006-03-11  Daniel Jacobowitz  <dan@codesourcery.com>

	* sysdeps/unix/sysv/linux/mips/nptl/fork.c: Correct path to
	i386/fork.c.

Index: sysdeps/unix/sysv/linux/mips/nptl/fork.c
===================================================================
RCS file: /cvs/glibc/ports/sysdeps/unix/sysv/linux/mips/nptl/fork.c,v
retrieving revision 1.1
retrieving revision 1.2
diff -u -p -r1.1 -r1.2
--- ports/sysdeps/unix/sysv/linux/mips/nptl/fork.c	28 Mar 2005 09:21:52 -0000	1.1
+++ ports/sysdeps/unix/sysv/linux/mips/nptl/fork.c	11 Mar 2006 16:20:03 -0000	1.2
@@ -1 +1 @@
-#include "../i386/fork.c"
+#include <sysdeps/unix/sysv/linux/i386/fork.c>
